### ISSUE TITLE
Another approach for "Put tag properties and methods on top"

### ISSUE THREAD

**Issue by amitguptagwl** [voorhoede/riotjs-style-guide#64]
I found a practical issue with this approach. When you tag file becomes longer it is difficult to track what is defined above. Specially when you refactor and change the method name.

So I use this way to create methods. Now if I need to refactor, I need to change only on one place. And my editor also doesn't complaint.

```js
tag.moveStart = function ( knotName ){ .. }
```
